oasst-shared/oasst_shared/schemas/protocol.py

```diff
@@ -256,9 +283,10 @@ class TextLabel(str, enum.Enum):
     slang = "slang"
 
 
-class TextLabels(BaseModel):
+class TextLabels(Interaction):
```

Comment:
this will add a `user` member to `TextLabels`, which has some effects on other parts and might break protocol, although I agree the change is necessary. Thus, a few other places need to be changed:
- the prompt repository -> store_text_labels, the user needs to be stored. this probably even requires a database schema migration to save the user (like in the other interactions)
- the LabelTextRequest endpoint (api/v1/text_labels) should be adjusted
- website and bot requests that depend on the text_labels endpoint should be adjusted

I realize this is a lot. if you feel like this might be a bit too much, just change this back to `BaseModel` and we'll take it from there

Reply:
Could you please make a notification in the #breaking-changes for this?

Reply:
Will do on merge